mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -827,6 +828,7 @@ public void getAccount(GetAccountRequest request, StreamObserver<GetAccountRespo
         mockCrnService.ensureProperAccountIdUsage(accountId);
         LOGGER.info("Get account: {}", accountId);
         Account.Builder builder = Account.newBuilder();
+        builder.addEntitlements(createEntitlement(CDP_CB_AWS_VERTICAL_SCALE));
```

Comment:
this way it is not configurable from the application.yml

Reply:
this is mock user management service. all other mock entitlements are being set this way

Reply:
Nope. Almos all of them are added dynamically based on application.yml. Please do so.

Reply:
done